src/librustc_typeck/check/method/confirm.rs

```diff
@@ -38,6 +38,11 @@ impl<'a, 'gcx, 'tcx> Deref for ConfirmContext<'a, 'gcx, 'tcx> {
     }
 }
 
+pub struct ConfirmResult<'tcx> {
+    pub callee: MethodCallee<'tcx>,
+    pub rerun: bool,
```

Comment:
Nit: can we give this a more informative name? It seems that it actually means that there was an illegal sized bound, right?